db/seeds/data/contracts/2021.csv

```diff
@@ -0,0 +1,69 @@
+provider_name,cohort_year,course_identifier,recruitment_target,per_participant,service_fee_installments,special_course,monthly_service_fee,funding_cap
```

Comment:
Do we need to use CSVs for the seed data? is it a case of the values are specific/constant for each lead provider/cohort?

Reply:
There are differences between cohort and courses. Using the existing seed csv from ecf will make sure we don't have unusual scenarios.

Reply:
hmmm I was hoping to avoid csvs as much as we can in NPQ, especially with the different cohorts. We were able to get away from this with the factories and seed data for statements as well as the statement populaters by making it more generic, is this something we can't replicate here? happy to have a chat